nautobot/dcim/models/racks.py

```diff
@@ -363,8 +371,9 @@ def get_reserved_units(self):
                 reserved_units[u] = r
         return reserved_units
 
-    def get_elevation_svg(
+    def get_elevation(
         self,
+        fileformat: Literal[RackElevationDetailRenderChoices.RENDER_CSV, RackElevationDetailRenderChoices.RENDER_SVG],
```

Comment:
Similarly, seems like a separate `get_elevation_csv` method would make more sense given that most of the parameters in the existing method are unused/irrelevant when exporting to CSV.